public/assets/html/file-detail.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
```

Comment:
This can also be removed, yes?

Reply:
Yes, these can be removed if unnecessary